File: abc/abc181/b.py
# https://atcoder.jp/contests/abc181/submissions/38146995

n = int(input())
ans = 0

a = 0
b = 0
for _ in range(n):
  a, b = map(int, input().split())
  ans += (b - a + 1) * (a + b) // 2

print(ans)

File: abc/abc181/b.cpp
// https://atcoder.jp/contests/abc181/submissions/27067572

#include <bits/stdc++.h>
using namespace std;
int main() {
  unsigned long long n, a, b, ans = 0; cin >> n;
  while(cin >> a >> b) ans += b * (b+1) / 2 - (a-1) * a / 2;
  cout << ans;
}
